### ISSUE TITLE
Home assistant restart fails with SyntaxError

### ISSUE THREAD

**Issue by fetzerch** [astrandb/miele#335]
### The problem

I noticed that after installing the component through HACS, I am no longer able to restart home assistant.
The problem seems to be caused by `pymiele` pulling in `asyncio` from PyPI and then on the next start homassistant loads this instead of `asyncio` from the standard library.

Could you please remove the dependency to `asyncio` from https://github.com/astrandb/pymiele/blob/main/setup.py#L46 and publish a new version? According to the documentation, this package is only meant to be used with Python 3.3 (https://pypi.org/project/asyncio/). On newer Python versions, it's fine to rely on `asyncio` being available in the standard library.

### Version of miele integration having the issue?

0.1.15

### Version of Home Assistant Core having the issue?

2023.9.3

### Anything in the logs that might be useful for us?

```txt
Traceback (most recent call last):
  File "/lsiopy/bin/hass", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/lsiopy/lib/python3.11/site-packages/homeassistant/__main__.py", line 171, in main
    args = get_arguments()
           ^^^^^^^^^^^^^^^
  File "/lsiopy/lib/python3.11/site-packages/homeassistant/__main__.py", line 81, in get_arguments
    from . import config as config_util
  File "/lsiopy/lib/python3.11/site-packages/homeassistant/config.py", line 20, in <module>
    from . import auth
  File "/lsiopy/lib/python3.11/site-packages/homeassistant/auth/__init__.py", line 4, in <module>
    import asyncio
  File "/config/lsiopy/lib/python3.11/site-packages/asyncio/__init__.py", line 21, in <module>
    from .base_events import *
  File "/config/lsiopy/lib/python3.11/site-packages/asyncio/base_events.py", line 296
    future = tasks.async(future, loop=self)
                   ^^^^^
SyntaxError: invalid syntax
```

**Comment by astrandb** [comment]
Thank you for reporting and tracking down this problem. I have not seen this problem before so there must be somehing specific to your installation that triggers the error.
I will follow your advice and adjust pymiele accordingly. However, I need your help to verify that the new version works before releasing a new version of the integration.   Please patch the requirement in `custom_components/miele/manifest.json` to `pymiele==0.1.7`

**Comment by fetzerch** [comment]
Thanks for the quick response. I can confirm that it works now as expected!

Would you mind also pushing the pymiele tags also to github? The last available is 0.1.5b1.

I'm not sure what's special about my setup. I'm using a containerized version of home assistant with Python 3.11 and have the Python virtual env on a volume so that a restart of home assistant is faster because it doesn't need to reinstall all Python packages. Maybe that's the reason others were not affected.

**Comment by astrandb** [comment]
Release tag v0.1.7 added to pymiele. I will update the requirement in the integration in next release.
